public/css/app.css

```diff
@@ -0,0 +1,310 @@
+/* body */
+
+body {
+    background-color: #ccc;
+}
+
+/* container */
+
+.container {
+    position: relative;
+        left: 50%;
+        transform: translateX(-50%);
+    margin-top: 10%;
+    padding: 0;
+    width: 80%;
+}
+
+/* outer circle */
+
+.outer-circle {
+    background-color: #222;
+    border-radius: 50%;
+    box-shadow: 0 0 3px #000;
+    padding-top: 100%;
+    width: 100%;
+}
+
+/* buttons */
+
+button {
+    border: none;
+    cursor: pointer;
+}
+button:active, button:focus {
+    outline: none !important;
+    border: none !important;
+    -webkit-touch-callout: none;
+    -webkit-user-select: none;
+    -khtml-user-select: none;
+    -moz-user-select: none;
+    -ms-user-select: none;
+    user-select: none;
+}
+
+/* quadrants */
+
+.qtr-circle {
+    box-shadow: 0 0 10px #000,
+                inset 0 0 50px #222;
+    height: 42.5%;
+    position: absolute;
+    width: 42.5%;
+}
+.tl {
+    background-color: green;
+    border-radius: 100% 0 0 0;
+    left: 5%;
+    top: 5%;
+}
+.tr {
+    background-color: #C71C1C;
+    border-radius: 0 100% 0 0;
+    left: 52.5%;
+    top: 5%;
+}
+.bl {
+    background-color: #C5C507;
+    border-radius: 0 0 0 100%;
+    left: 5%;
+    top: 52.5%;
+}
+.br {
+    background-color: #0808C1;
+    border-radius: 0 0 100% 0;
+    left: 52.5%;
+    top: 52.5%;
+}
+
+/* middle circle */
+
+.middle-circle {
+    background-color: #222;
+    border-radius: 50%;
+    box-shadow: 0 0 50px #222;
+    height: 50%;
+    position:absolute;
+        top: 25%;
+        left: 25%;
+    width: 50%;
+
+}
+
+ /* inner circle */
+
```

Comment:
If the inner-circle is really just the game controls, then you can probably call it something like that.  You don't want to tie your code to a visual position as much as possible, since positions can change.  Start thinking like this now, the less dependencies your code has the better.  Choosing a good balance between being flexible and being descriptive.